src/nspawn/nspawn.c

```diff
@@ -448,6 +456,57 @@ static int parse_argv(int argc, char *argv[]) {
                         arg_settings_mask |= SETTING_NETWORK;
                         break;
 
+                case ARG_NETWORK_VETH:
+                        {
+                                _cleanup_strv_free_ char **a = NULL;
+                                int l;
+
+                                if (strempty(optarg))
+                                        return log_error_errno(r, "--network-veth= arguments missing .");
+
+                                p = optarg;
+                                for (;;) {
+                                        _cleanup_free_ char *t = NULL;
+
+                                        r = extract_first_word(&p, &t, ":", 0);
```

Comment:
You probably want to use `EXTRACT_DONT_COALESCE_SEPARATORS` here,
since otherwise you could pass `--network-veth=:::` and it will happen to treat that as `-n` because there's no explicit check for getting 0 options so it falls through to doing nothing, and the `isempty(optarg)` check doesn't think it's an empty argument.